bot/__init__.py

```diff
@@ -30,5 +31,8 @@ def start_bot():
     log.info("setuping cogs...")
     register_all_cogs(BOT)
 
+    log.info("setuping events...")
```

Comment:
Let's decide which quotes to use. Because in one pull request single and double quotes.